### ISSUE TITLE
A11y: Tag follow/unfollow buttons should be buttons, not anchors

### ISSUE THREAD

**Issue by RyKilleen** [thepracticaldev/dev.to#126]
# _TASK_
### Request or User Story
The `.follow-action-button` buttons should be button elements, to better describe their purpose. They aren't used as page anchors and never reference anywhere else.
